packages/simcore-sdk/src/simcore_sdk/node_ports_common/config.py

```diff
@@ -1,7 +1,6 @@
 """Takes care of the configurations.
 """
 import os
-from typing import Dict, Type
 
 # required configurations
 STORAGE_ENDPOINT: str = os.environ.get("STORAGE_ENDPOINT", default="undefined")
```

Comment:
MINOR:
- perhaps we should start using proper Settings at some point to capture environs
- since there is not more version 1 and 2, why not put node_ports_common in node_ports_v2?

Reply:
agreed. will do in the next iteration